ui/src/components/Preview/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import Markdown from '@mozilla-frontend-infra/components/Markdown';
+import ListItem from '@material-ui/core/ListItem';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import TextField from '@material-ui/core/TextField';
+
+export default class Preview extends Component {
+  state = {
+    value: 0,
+  };
+
+  handleChange = (event, value) => {
+    this.setState({ value });
+  };
+
+  render() {
+    const { description, onDescriptionChange, placeholder } = this.props;
+    const { value } = this.state;
+
+    return (
+      <div>
+        <ListItem>
+          <Tabs value={value} onChange={this.handleChange}>
+            <Tab label="Description" />
+            <Tab label="Preview" />
+          </Tabs>
+        </ListItem>
+        <ListItem>
+          {value === 0 && (
```

Comment:
Wrap the content with a div:

```jsx
<div className={classNames(classes.tabContent, [classes.markdownWrapper]: value === 1)}>
   {value === 0 && // ... }
  {value === 1 && // ... }
</div>
```

```jsx
@withStyles(theme => {
  const borderColor =
    theme.palette.type === 'light'
      ? fade(THEME.BLACK, 0.23)
      : fade(THEME.WHITE, 0.23);

  return {
    tabContent: {
      marginTop: theme.spacing.unit,
      padding: theme.spacing.unit,
    },
    markdownContainer: {
      borderColor,
      borderWidth: 1,
      borderRadius: theme.shape.borderRadius,
      borderStyle: 'solid',
    },
  };
})
```

`THEME.BLACK` is probably not defined. You will need to add it.